ygot/diff_test.go

```diff
@@ -1576,6 +1576,9 @@ func TestDiff(t *testing.T) {
 		t.Run(tt.desc+"Diff", func(t *testing.T) {
 			got, err := Diff(tt.inOrig, tt.inMod, tt.inOpts...)
 			testDiffSingleNotif(t, "Diff", got, err)
+			if diffout := FormatDiff(got); diffout == "" {
```

Comment:
Shall we add one test case that shows that the string is what we expect it to be? I know there is no contract around it, but it seems like it might be a reasonable example?

Reply:
Done.